src/main/java/seedu/duke/Main.java

```diff
@@ -58,12 +58,7 @@ public static void main(String[] args) throws IOException, TaskManagerException,
                 }
             }
             printWeek = true; // Reset flag for the next iteration
-            System.out.println("Enter 'next' for next week, 'prev' for previous week, " +
-                    "'add' to add a task, " +
-                    "'update' to edit a task, " +
-                    "'delete' to delete a task, " +
-                    "'month' to display the month view, " +
-                    "or 'quit' to quit:");
+
             System.out.println("Enter help to learn commands");
```

Comment:
This does make the overall user experience cleaner and more straightforward, with help available solely through the 'help' command!